_ext/mixcloud.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Mopidy-Mixcloud
+type: backend
+dev:
+  github: unusualcomputers/unusualcomputers/tree/master/code/mopidy/mopidymixcloud 
+dist:
+  pypi: Mopidy-Mixcloud
+
+---
+
+A backend for playing and browsing feeds from [Mixcloud](https://mixcloud.com).
+
+For more information see [github](https://github.com/unusualcomputers/unusualcomputers/blob/master/code/mopidy/mopidymixcloud/README.md).
```

Comment:
```suggestion

For more information see [README](https://github.com/unusualcomputers/unusualcomputers/blob/master/code/mopidy/mopidymixcloud/README.md).
```

I'd probably not bother with this extra line, every single extension has more information in its README and a handy link to the repo will be provided automatically. In which case:
```suggestion
```